kano_settings/home_screen.py

```diff
@@ -36,165 +20,102 @@ class UnknownScreenError(Exception):
 
 
 class HomeScreen(Gtk.Box):
+    _DISPLAYED_SCREENS = SCREENS.get_screens_on_home()
 
-    names = ["Keyboard", "Mouse", "Audio", "Display", "Wifi", "Overclocking",
-             "Account", "Appearance", "Font", "Advanced", "About",
-             "Notifications"]
-    custom_info = ["Keyboard-country-human", "Mouse", "Audio", None, None,
-                   "Overclocking", None, "Wallpaper", "Font"]
+    def __init__(self, win, screen_number=None, screen_name=None):
```

Comment:
Cool. It's all right if we're removing it anyway :-).

:shipit: 

Certainly an improvement to the codebase!